apps/codeforafrica/src/lib/index.js

```diff
@@ -655,7 +661,56 @@ function getProjectTags(options = { includeAll: true }) {
   return tags;
 }
 
-function getHomePageStaticProps() {
+function paginateResults(items, page, pageSize) {
+  // We need to initialize to null for serialization.
+  let count = null;
+  let results = [];
+  let pageNumber = null;
+  let pageSizeNumber = null;
+  if (items?.length) {
+    // Need to ensure page, pageSize are numbers and not strings
+    pageNumber = Number.parseInt(page, 10) || 1;
+    pageSizeNumber = Number.parseInt(pageSize, 10) || 6;
+    count = Math.ceil(items.length / pageSizeNumber);
+    results = items.slice(
+      (pageNumber - 1) * pageSizeNumber,
+      pageNumber * pageSizeNumber
+    );
+  }
+  return {
+    pagination: {
+      count,
+      page: pageNumber,
+      pageSize: pageSizeNumber,
+    },
+    results,
+  };
+}
+
+export async function getStories(options) {
+  const {
+    tag: originalTag,
+    page,
+    "page-size": pageSize = 10,
+    q,
+  } = options || {};
+  const tag = originalTag || "stories";
```

Comment:
I think we're mixing **normal** tags (`Kenya`, `Misinformation`, etc.) with **primary** tags (`Stories` and `Opportunities`). The tag here should be one of the _normal_ tags (or All) since the method name specifically says these are stories.